app/models/paper.rb

```diff
@@ -23,11 +25,11 @@ class Paper < ActiveRecord::Base
     end
 
     event :start_meta_review do
-      transitions :from => :submitted, :to => :review_pending, :after => :create_meta_review_issue
+      transitions :from => :submitted, :to => :review_pending, :if => :create_meta_review_issue
```

Comment:
This conditional makes sure that the AASM transition only happens _if_ the `create_meta_review_issue` method returns `true`